guides/faq.rst

```diff
@@ -83,8 +83,8 @@ Select the firmware binary and finally press "Flash ESP".
     are connected correctly if flashing using a USB to UART bridge. For some devices you need to
     keep pressing the BOOT button until flashing has begun (ie. Geekcreit DOIT ESP32 DEVKIT V1).
 
-Help! Something's not working!1!
---------------------------------
+Help! Something's not working!!
```

Comment:
Well this one was kinda intentional but ok :)

Reply:
Okay. I didn't get the intention... just looked like a mistake to me.